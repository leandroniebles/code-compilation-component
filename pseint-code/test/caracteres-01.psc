proceso car
  leer varíable_1;
  leer variable²;
  leer var_á;
  leer var_é;
  leer var_í;
  leer var_ó;
  leer var_ú;
  leer var_Ú;
  leer var_Ó;
  leer var_É;
  leer var_Í;
  leer var_Á;
  leer var_Ü;
  leer var_ü;
  leer var_ñ;
  leer var_Ñ;
finproceso