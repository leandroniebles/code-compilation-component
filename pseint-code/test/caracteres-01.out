=== Line 3: SynError 68
=== Line 3: SynError 65
